decayAmplitude/waveDescription.cc

```diff
@@ -63,70 +63,103 @@ using namespace rpwa;
 namespace {
 
 
-	typedef boost::shared_ptr<libconfig::Config> configPtr;
+	typedef boost::shared_ptr<Config> configPtr;
 
 
-	const std::string binBorders = "bins";
-	const std::string expandName = "expand";
-	const std::string expandTypeName = "type";
-	const std::string expandNameName = "name";
+	const string binBorders     = "bins";
+	const string degreeName     = "degree";
+	const string mMinName       = "mMin";
+	const string mMaxName       = "mMax";
+	const string realCoeffName  = "realCoefficients";
+	const string imagCoeffName  = "imagCoefficients";
```

Comment:
I don't think that this is a good place to store option names for some mass dependences.
Also, some of them seem not to be used.

Since the configuration parameters of an amplitude are in some private and specific to the respective amplitude, it would be better to add a member function to each mass dependence that retrieves the necessary information from the libconfig Setting.